Resources/Private/Templates/View3D/Main.html

```diff
@@ -13,7 +13,7 @@
       xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
       data-namespace-typo3-fluid="true">
 
-    <p id="DFG_3DViewer" 3d="{url}" class="tx-dlf-view-3d"></p>
+    <p id="DFG_3DViewer" 3d="{url}" 3d="{proxy}" class="tx-dlf-view-3d"></p>
```

Comment:
Is it intentional, that both the plain URL and the Proxy-URL use the same attribute name?

Reply:
Do we even need the plain URL when the proxy should be used?

Reply:
> Is it intentional, that both the plain URL and the Proxy-URL use the same attribute name?

No, in later changes it is adjusted to _proxy_. The original one (_3d_) is used when proxy URL needs to be modified with the corrected 3D model.

Reply:
OK, thanks for the explanation!